packages/terra-time-input/package.json

```diff
@@ -30,7 +30,7 @@
     "classnames": "^2.2.5",
     "keycode-js": "^2.0.1",
     "prop-types": "^15.5.8",
-    "terra-button-group": "^3.0.0",
+    "terra-button-group": "3.37.0",
```

Comment:
Why did we lock this down?

Reply:
The inputs appeared in a vertical order in the screenshots, will be reverted once this [issue](https://github.com/cerner/terra-core/issues/2947) is fixed.

Reply:
> The inputs appeared in a vertical order in the screenshots

Are you saying the inputs have the wrong css alignment? instead of in-line they are appearing as stacked? The issue you linked is for hover styles?

Reply:
Which examples or screenshots are you seeing the stacked inputs? I'm not able to recreate it on v3.38.0.

Reply:
looks like travis failed when I opened it up.  Something's going on but I'm curious why we think it might be due to hover styles?

Reply:
<img width="171" alt="TimeInput" src="https://user-images.githubusercontent.com/42207428/81811270-fee42980-9541-11ea-8158-c3d5f2666284.png">

Multiple tests are failing [this](https://github.com/cerner/terra-framework/blob/202716959067f01c1621679a483557781a2f7580/packages/terra-time-input/tests/wdio/time-input-spec.js#L387) is one of those